src/applications/dhp-connected-devices/components/DeviceDisconnectionCard.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const DeviceDisconnectionCard = ({ vendor, onClickHandler }) => {
+  return (
+    <div className="connect-device">
+      <h3 className="vads-u-margin-y--0">{vendor}</h3>
```

Comment:
I'm looking at the wire frames, and it looks like we're just missing `- Connected` after the vendor name. So, in this section it should look like:
**Vendor** - Connected
(button) Disconnect

I also just noticed a small difference between the `Disconnect` button and the wireframe. The wireframe has the background of the button white, whereas it looks grey here.

Reply:
@yonidoronpeters

Reply:
Good catch! will update to "Fitbit - Connected"

The button we use has a clear background. I don't see a button with a white background in [Storybook](https://design.va.gov/storybook/?path=/docs/components-buttons--primary#secondary-buttons) so wonder if the white background was by design or by mistake. @toddstanich thoughts on the button?

Reply:
Pushing the `- Connected` updated as well as SASS for a button with a white background. It's commented out though pending @toddstanich's thoughts.

Reply:
I also noticed a discrepancy in the page title which I updated.

Reply:
We can leave the disconnect button as is. It looks like it is an error in the Sketch design library and the component does not have a white BG.